crates/full-node/sov-sequencer/src/preferred/mod.rs

```
@@ -277,21 +279,25 @@ where
             _runtime: PhantomData,
             config: config.clone(),
             shutdown_receiver: shutdown_receiver.clone(),
-            shutdown_sender,
+            shutdown_sender: shutdown_sender.clone(),
             tx_queue_id,
             stop_at_rollup_height,
             test_only_state_update_notification_sender: broadcast::channel(100).0,
         });
 
-        // Launch replica sync task only for replicas
-        // This will block until the currently stored batches in the DB are replayed onto the
-        // state, then yield when it switches to processing postgres events live.
-        // This is necessary to prevent conflicts with the update_state task.
+        // Launch replica sync task only for replicas.
         if config.sequencer_kind_config.is_replica {
-            //handles.push(
-            //    spawn_replica_sync_task(seq.clone(), shutdown_receiver.clone(), latest_db_event_id)
-            //        .await,
-            //);
+            if let Some(postgres_connection_string) =
```

Should we throw an error if `config.sequencer_kind_config.is_replica && postgres_connection_string.is_none()`?
Small nit since failover will make this redundant but maybe worth adding for now while we've got replicas without failover

---

Will fix in the next PR